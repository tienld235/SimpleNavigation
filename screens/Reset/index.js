import ResetScreen from "./ResetScreen";

export default ResetScreen;