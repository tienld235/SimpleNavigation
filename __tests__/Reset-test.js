import "react-native";
import React from "react";
import Reset from "../screens/Reset";
import renderer from "react-test-renderer";

test("Reset Screen Snapshot", () => {
  const snap = renderer.create(<Reset />).toJSON();
  expect(snap).toMatchSnapshot();
});
