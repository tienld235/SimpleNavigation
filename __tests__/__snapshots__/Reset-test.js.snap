// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset Screen Snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is the ResetScreen.
  </Text>
</View>
`;
